src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -233,6 +236,16 @@ private void TryWriteHostJson(string filePath, JObject content)
                 }
             }
 
+            private void TryAddBundleConfiguration(JObject content)
```

Comment:
I think this is correct (since you're modifying the input content through `Add` and not reassigning (like we do in `InitializeHostConfig`, line 167, which is a bug).

Do you mind explicitly returning the modified JObject content though, just to be explicit that the value was modified? It seems like this is a bug waiting to happen

Reply:
Done..